### ISSUE TITLE
[Adaptive Cards > WeatherCompact.Json] - Voice over focus is moving separately on '46' and 'degree F' in 'WeatherCompact.json' card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#44]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'WeatherCompact.json' under 'Scenarios' list item and activate it.
7.  Card will be appear.
8. Navigate to '46' and 'degree F'
9. Observe voice over issue.
 
**Actual Result:**
Voice over focus is moving separately on '46' and 'degree F' in 'WeatherCompact.json' card.

**Expected Result:**
Voice over focus should move together on '46' and 'degree F' in 'WeatherCompact.json' card.
Voice over should announce as '46 degree F'.

**User Experience:**
Voice over user will be confused if proper reading order is not followed by voice over.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/b7ee393b-55f2-4596-af8a-e93dcd2e82a8

**Comment by GabrielMedAlv** [comment]
The card, by definition, contains 2 different TextBlocks for value and degree type. I can't provide a general solution for this because is correct to show the accessibility separately. Solution for this will be more on card definition.

**Comment by vagpt** [comment]
Same issue is repro on Android Platform as well.
